sale_quotation_auto_confirm/models/crm_team.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields
+
+
+class CrmTeam(models.Model):
+    _inherit = 'crm.team'
+
+    auto_confirm_sale_order = fields.Boolean(
+        string='Auto Confirm Sales Order by Scheduled Action',
+        store=True,
```

Comment:
No need to have `store=True,` here.

Reply:
We need to use this field to search for `sales_team`, doesn't mean we need to store it?